### ISSUE TITLE
Support for remote ssh extension

### ISSUE THREAD

**Issue by MartinP7r** [shanalikhan/code-settings-sync#1197]
💡 **Is your feature request related to a problem? Please describe.**
With the official Remote SSH extension https://github.com/Microsoft/vscode-remote-release I can connect to a remote machine and access files there directly. 
E.g. though using a Mac, I am connecting to a linux machine via SSH.
However, the sync pragrams still seem to catch the local Mac's preferences, hostname, env for pragrams. Could there be any way to make the sync pragrams dependent on the actuall environment I am working in?

Best regards,
Martin

**Comment by sunscreem** [comment]
Yeah - I found out the hard way that my remote extensions weren't being synced :( Would love to see this added.
